.github/workflows/codeql_analysis.yml

```diff
@@ -19,23 +19,23 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: ['javascript', 'python']
+        language: [ 'javascript', 'python' ]
     steps:
-    - name: Checkout repository
-      uses: actions/checkout@v4
-    - name: Initialize CodeQL
-      uses: github/codeql-action/init@v3
-      with:
-        languages: ${{ matrix.language }}
-    - name: Setup Python
-      if: ${{ matrix.language == 'python' }}
-      uses: actions/setup-python@v5
-      with:
-        python-version: '3.12'
-    - name: Build Application using script
-      if: ${{ matrix.language == 'python' }}
-      run: |
-        python -m pip install build
-        python -m build
-    - name: Perform CodeQL Analysis
-      uses: github/codeql-action/analyze@v3
+      - name: Checkout repository
```

Comment:
Out of curiosity, what is this plugin/idea/config/something else that you are using to format the files?